app/controllers/api/v1/allocation_events_controller.rb

```diff
@@ -3,15 +3,21 @@
 module Api
   module V1
     class AllocationEventsController < ApiController
-      before_action :validate_params, only: :create
-      after_action :send_move_notifications, only: :create
+      before_action :validate_params, only: :events
+      after_action :send_move_notifications
 
-      def create
+      def cancel
+        allocation.cancel(cancellation_details)
+
```

Comment:
currently in the move event controller, we are replaying the event logs using `EventLog::MoveRunner.new(move).call`. Here although we are cancelling an allocation, it is also cancelling moves as well. Is there a way we can integrate those into the move runner?

Reply:
I don't really know what this is or whether it's still relevant to the new approach. Maybe we can discuss?

Reply:
Added new event log goodness - see new `Eventable` concern.